src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1431,9 +1431,11 @@ void scrollElementToCenterAndClick(WebElement element) {
     }
 
     /**
-     * Scrolls element to first cell in spreadsheet and clicks on it.
+     * Scrolls element to first cell in enroll spreadsheet and clicks on it.
```

Comment:
why only enroll spreadsheet?

Reply:
Same as this https://github.com/TEAMMATES/teammates/pull/8884#discussion_r195302082

Reply:
This is like a helper function. I think it would be better to make it as general as possible, so that it can be reused by other spreadsheets, if the needs arise.